package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "login-forms",
+  "version": "1.0.0",
+  "description": "A collection of templates for the CMS login screens",
+  "scripts": {
+    "build": "yarn && yarn lint && yarn test && NODE_ENV=production webpack -p --bail --progress",
+    "watch": "yarn && NODE_ENV=development webpack --watch --progress",
+    "css": "WEBPACK_CHILD=css npm run build",
+    "lint": "eslint client/src && sass-lint -v",
+    "test": "jest",
+    "coverage": "jest --coverage",
+    "js-fix": "eslint client/src --fix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/silverstripe/login-forms.git"
+  },
+  "author": "SilverStripe Ltd",
+  "license": "BSD-3-Clause",
+  "bugs": {
+    "url": "https://github.com/silverstripe/login-forms/issues"
+  },
+  "homepage": "https://github.com/silverstripe/login-forms#readme",
+  "dependencies": {
+  },
+  "devDependencies": {
+    "@silverstripe/webpack-config": "^0.4.0",
```

Comment:
I think we're up to `^0.14.0`.

Reply:
Of course we are, good spotting! Will update...